programs/zstdcli.c

```diff
@@ -957,7 +962,10 @@ int main(int const argCount, const char* argv[])
                 case 't': operation=zom_test; argument++; break;
 
                     /* destination file name */
-                case 'o': nextArgumentIsOutFileName=1; lastCommand=1; argument++; break;
+                case 'o':
+                    NEXT_FIELD(outFileName);
+                    if (!strcmp(outFileName, "-")) outFileName = stdoutmark;
```

Comment:
ah yes, good point !
`-o -` will fail, since the second `-` will be confused as the beginning of a new command,
so the macro will complain about missing argument, display an error message and then `CLEAN_RETURN()`.

I'm not sure if we need to support this. Chances are, we don't, so it's just better to remove this line.